courses/machine_learning/deepdive2/feature_engineering/solutions/Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2721 @@
+{
```

Comment:
<p>Are you writing this for colab too? Or is there a reason you have the try statement? My team has been checking for the proper tensorflow version through bash like this:</p><p> </p><p>!python3 -m pip freeze | grep 'tensorflow==2\|tensorflow-gpu==2' || \</p><p>python3 -m pip install tensorflow==2.0.0</p><p> </p><p>which can be added to your !pip install sklearn cell above.</p><p> </p><p>I also noticed your using just keras instead of tensorflow.keras. Any reasons why?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<ol><li> I initially started writing this in colab and then switched to the AI platform.  Removed the "try/except" check.</li><li>I added the bash statements to the !pip install sklearn cell.  <a href="  https://docs.google.com/presentation/d/15SQKH0fgb5nzsfkVRGAXNtsZqyG739vzCI22zEc9Ilw/edit#slide=id.g479e29862b_0_1" target="_blank">See slide #1</a> for the output - cannot find TensorFlow distribution 2.0.  This is strange, as I am using a TensorFlow 2.0 instance.</li></ol><ul><li>Keras vs. tensorflow.keras -  there are tf.keras in the code; but I did not know I needed to import the formal library - so, I added "import tensorflow.keras".</li></ul><p> </p>

Reply:
Were you still having trouble with this one? I noticed from your notebook, it found the beta version of tensorflow, so if it's still throwing errors at you, maybe a new notebook instance with the newest version of tensorflow will fix it?

Reply:
<p>Hi Danielle - I created a new Notebook instance.</p>